### PR TITLE
feat: reduced marketplace bond in KSM

### PR DESCRIPTION
**Pull request by lumir-mrkva**

### CODE REVIEW

File: integration-tests/src/nft_marketplace.rs [hunk 1]
```diff
@@ -0,0 +1,136 @@
+#![cfg(test)]
```
Comment (green-jay) [inline integration-tests/src/nft_marketplace.rs:1]: if we have pallet specific tests we could separate them one level deeper into a folder

File: integration-tests/src/cross_chain_transfer.rs [hunk 2]
```diff
@@ -31,24 +31,24 @@ fn transfer_from_relay_chain() {
...
-			13 * BSX
+			13 * UNITS
 		);
 	});
 
 	Basilisk::execute_with(|| {
 		assert_eq!(
 			basilisk_runtime::Tokens::free_balance(1, &AccountId::from(BOB)),
-			10028 * BSX / 10 // 3 BSX - fees
+			10028 * UNITS / 10 // 3 BSX - fees
```
[the remark this answers is not in the document]
Comment (lumir-mrkva) [reply integration-tests/src/cross_chain_transfer.rs:47]:
```suggestion
			10028 * UNITS / 10
```

File: integration-tests/src/cross_chain_transfer.rs [hunk 3]
```diff
@@ -126,18 +126,18 @@ fn transfer_from_hydra() {
...
-			200 * BSX - 3 * BSX
+			200 * UNITS - 3 * UNITS
 		);
 	});
 
 	Basilisk::execute_with(|| {
 		assert_eq!(
 			basilisk_runtime::Tokens::free_balance(1, &AccountId::from(BOB)),
-			10028 * BSX / 10 // 3 * BSX - fees
+			10028 * UNITS / 10 // 3 * BSX - fees
```
[the remark this answers is not in the document]
Comment (lumir-mrkva) [reply integration-tests/src/cross_chain_transfer.rs:136]:
```suggestion
			10028 * UNITS / 10
```

File: integration-tests/src/cross_chain_transfer.rs (hunk 2 above)
Comment (lumir-mrkva) [inline integration-tests/src/cross_chain_transfer.rs]:
```suggestion
			10028 * UNITS / 10
```

**Pull request merged**